### ISSUE TITLE
Statistics page broken

### ISSUE THREAD

**Issue by beepsoft** [keeps/roda#1047]
I tried the latest Docker version  of RODA (https://hub.docker.com/r/keeps/roda/) as well as the online demo (https://demo.roda-community.org/?locale=en#administration/statistics) and I feel that the Statistics page is broken. It has a lot of empty rectangles, which I suppose, should have different charts in them. The browser's console shows errors like this, when loading the page:

    theme?resource_id=js/statistics.js:345 Uncaught TypeError: Cannot read property 'values' of null

    at facetCommonChartOptions (theme?resource_id=js/statistics.js:345)

    at facetPieChartOptions (theme?resource_id=js/statistics.js:279)
  
    at viewFacetChartCallback (theme?resource_id=js/statistics.js:195)
  
    at Object.<anonymous> (theme?resource_id=js/statistics.js:126)
  
    at j (jquery-1.11.3.min.js:2)
  
    at Object.fireWith [as resolveWith] (jquery-1.11.3.min.js:2)
  
    at x (jquery-1.11.3.min.js:5)
  
    at XMLHttpRequest.b (jquery-1.11.3.min.js:5)

Another - maybe unrelated problem  - is that when trying to download contents of any of the boxes (eg. "Distribution of description levels (top 10)") it downloads a file with .csv extension, although it actually contains XML.

**Comment by jmaferreira** [comment]
Mostly likely there is no data to show on those charts as the repository doesn't have enough AIPs or events to report.

Is there any instance where data exist in the repository and no chart is being displayed, e.g. there an ingest process but not chart about it?

**Comment by hsilva-keep** [comment]
Dear @beepsoft, first of all thank you for reporting an issue that you've encountered. 
But, in what concerns the issue you've described, can you please say what browser are you using? Because I've accessed it via Chrome & Firefox and it is working as expected (with some charts just showing an chart placeholder because no data exists for that matter as @jmaferreira already explained).

![captura de ecra de 2017-09-01 09 43 04](https://user-images.githubusercontent.com/1315759/29962228-5a9ded2a-8efa-11e7-92b7-67fd78163c4e.png)

Also, I've tried as well downloading the chart values and I've received a file with .csv extension containing csv data. Can you also provide the content you say is xml?

**Comment by beepsoft** [comment]
Hi,

thanks for the hints. I originally tried with Chrome and the Statistics page looked like this:

<img width="1476" alt="screen shot 2017-09-01 at 10 56 52" src="https://user-images.githubusercontent.com/1926265/29962689-53e7ad7c-8f04-11e7-85b5-5edab28cbf9d.png">

Now I tried with FF and it was OK, then I tried again in Chrome in a private window and then it worked too, so it must have been one of my extensions that messed up the content, sorry for the false alarm.

Regarding the downloaded statistics content: when I download eg. the "Distribution of representation types" I got this: (it downloaded as representation-types.csv, but for github I needed to rename to representation-types.txt to be able to upload, but nevertheless it contains XML): 

[representation-types.csv](https://github.com/keeps/roda/files/1269698/representation-types.txt)

<img width="1145" alt="screen shot 2017-09-01 at 11 05 06" src="https://user-images.githubusercontent.com/1926265/29962975-7463df16-8f05-11e7-8685-0ccb3ddbe8c1.png">

I got the same result in Chrome, Chrome private window, Firefox.
